kotsadm/pkg/store/ocistore/session_store.go

```diff
@@ -88,6 +83,11 @@ func (s OCIStore) GetSession(id string) (*sessiontypes.Session, error) {
 		return nil, errors.Wrap(err, "failed to unmarshal session")
 	}
 
+	// sessions created before this change will not have IssuedAt
+	if session.IssuedAt.IsZero() {
+		session.IssuedAt = session.ExpiresAt.AddDate(0, 0, -14)
```

Comment:
this won't have any effect though

Reply:
sure but id like to use this in the future to expire sessions. say an admin reconfigures the identity provider. you can invalidate all sessions with issued at prior to reconfiguration.